manifest/armv7l/i/imagemagick7.filelist

```diff
@@ -450,14 +449,14 @@
 /usr/local/lib/libMagickWand-7.Q16HDRI.so
 /usr/local/lib/libMagickWand-7.Q16HDRI.so.10
 /usr/local/lib/libMagickWand-7.Q16HDRI.so.10.0.1
-/usr/local/lib/perl5/5.38.1/core_perl/perllocal.pod
-/usr/local/lib/perl5/5.38.1/vendor_perl/Image/Magick.pm
-/usr/local/lib/perl5/5.38.1/vendor_perl/Image/Magick/Q16HDRI.pm
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/.packlist
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/Magick.so
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/Q16HDRI/Q16HDRI.so
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/Q16HDRI/autosplit.ix
-/usr/local/lib/perl5/5.38.1/vendor_perl/auto/Image/Magick/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/Image/Magick.pm
+/usr/local/lib/perl5/armv8l-linux-thread-multi/Image/Magick/Q16HDRI.pm
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/.packlist
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Magick.so
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Q16HDRI/Q16HDRI.so
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/Q16HDRI/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/auto/Image/Magick/autosplit.ix
+/usr/local/lib/perl5/armv8l-linux-thread-multi/perllocal.pod
```

Comment:
This path name does not match the architecture.

Reply:
Can you see if it works on arm machines?

Armv8l is what armv7l reports when loaded on an aarch64 kernel.

Reply:
Also the entire `imagemagick7` autotools build is ornery and basically impossible to debug, so I don't know how to fix this short of building it in a qemu container which might take a long time...

Reply:
I'll just merge it and we can fix if it breaks.